components/MapContainer.vue

```diff
@@ -74,7 +74,7 @@ const emit = defineEmits(['map-moved'])
 
 const handlePinClick = (resultId: string) => {
     //Let's show the result details
-    searchResultsStore.setActiveSearchResult(resultId)
+    searchResultsStore.setActiveFacility(resultId)
```

Comment:
I'm complating if this should still be activeSearchResult, even though it's a facility.
In the future, we've been discussing if the search results might be  facilities instead of hp's

Reply:
after thinking more, activeFacility is probably ok